addons/filesystem/stringtable.xml

```diff
@@ -121,6 +121,60 @@
                 <French>Si le contenu du fichier est une fonction ou un code exécutable depuis le terminal</French>
                 <Italian>Se il contenuto del file è una funzione o codice che può essere eseguito dal terminale</Italian>
             </Key>
+            <Key ID="STR_AE3_Filesystem_Config_IsEncryptedDisplayName">
```

Comment:
"Is encrypted?" suggests, that the user input is already encrypted. I think the usage of just the verb "encrypt?" would
be better

Reply:
Yeah, you are right. But I don't really like questionmarks in UI. So, I think I will try the imperative form: encrypt content (Or something like that)